### ISSUE TITLE
feat: Simple Object Detection from Webcam

### ISSUE THREAD

**Issue by heysaiyad** [heysaiyad/dev-toolkit#203]
Create a tool that uses a user's webcam and a pre-trained model from TensorFlow.js (like COCO-SSD) to detect and label common objects in the video stream in real-time.
